sql/mob_skill_lists.sql

```diff
@@ -3932,7 +3932,53 @@ INSERT INTO `mob_skill_lists` VALUES ('Avatar_Ifrit_WTB',2013,847); -- Waking th
 
 -- Next available ID: 2014
 
--- ------------------------------------------------------------
+INSERT INTO `mob_skill_lists` VALUES ('Supernal_Chapuli',2015,2945); -- natures_meditation
+INSERT INTO `mob_skill_lists` VALUES ('Supernal_Chapuli',2015,2946); -- sensilla_blades
+INSERT INTO `mob_skill_lists` VALUES ('Supernal_Chapuli',2015,2947); -- tegmina_buffet
+INSERT INTO `mob_skill_lists` VALUES ('Supernal_Chapuli',2015,2948); -- sanguinary_slash
+
+INSERT INTO `mob_skill_lists` VALUES ('Supernal_Chapuli',2016,2949); -- orthopterror
+
+INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym',2017,2950); -- tempestuous_upheaval
+INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym',2017,2952); -- blackout
+
+INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym_low_50',2018,2950); -- tempestuous_upheaval
+INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym_low_50',2018,2952); -- blackout
+INSERT INTO `mob_skill_lists` VALUES ('Unfettered_Twitherym_low_50',2018,2953); -- smouldering_swarm (Below 50%)
+
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1951); -- magma_fan
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1952); -- erratic_flutter
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1953); -- proboscis
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1954); -- erosion_dust
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1955); -- exuviation
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2019,1956); -- fire_break
+
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1951); -- magma_fan (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1952); -- erratic_flutter (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1953); -- proboscis (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1954); -- erosion_dust (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1955); -- exuviation (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,1956); -- fire_break (Below 50%)
+INSERT INTO `mob_skill_lists` VALUES ('Taxet',2020,3044); -- benumbing_blaze (Below 50%)
+
+INSERT INTO `mob_skill_lists` VALUES ('Transcendent_Scorpion',2021,353); -- death_scissors
+INSERT INTO `mob_skill_lists` VALUES ('Transcendent_Scorpion',2021,354); -- wild_rage
+INSERT INTO `mob_skill_lists` VALUES ('Transcendent_Scorpion',2021,356); -- sharp_strike
+
+INSERT INTO `mob_skill_lists` VALUES ('Transcendent_Scorpion',2022,353); -- death_scissors (Below 50%)
```

Comment:
Transcendent_Scorpion -> Transcendent_ScorpionTranscendent_Scorpion_low_50, like you've done with Unfettered_Twitherym